src/core/actions/installations.js

```diff
@@ -31,7 +31,6 @@ export const setInstallError = ({
   error,
 }: SetInstallErrorParams): SetInstallErrorAction => {
   invariant(guid, 'guid is required');
-  invariant(error, 'error is required');
```

Comment:
This is needed because an ancient test case was explicitly testing an install error with no error message.